src/serac/physics/base_physics.hpp

```diff
@@ -363,19 +363,29 @@ class BasePhysics {
 
   /**
    * @brief Set the loads for the adjoint reverse timestep solve
+   * @param string_to_dual An unorder map from the state field name string to the finite element adjoint/dual load
+   * The adjoint load is d(qoi)/d(state)
    */
-  virtual void setAdjointLoad(std::unordered_map<std::string, const serac::FiniteElementDual&>)
+  virtual void setAdjointLoad(std::unordered_map<std::string, const serac::FiniteElementDual&> string_to_dual)
   {
-    SLIC_ERROR_ROOT(axom::fmt::format("Adjoint analysis not defined for physics module {}", name_));
+    if (string_to_dual.size()) {
```

Comment:
If you're checking whether the map is populated here, I suggest using the `empty()` method on STL containers, because the intent is clearer.